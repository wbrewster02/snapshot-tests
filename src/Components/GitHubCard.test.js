import renderer from 'react-test-renderer'
import GithubCard from './GitHubCard'

test('renders a snapshot', () => {
    const card = renderer.create(<GithubCard/>).toJSON()
    expect(card).toMatchSnapshot()
  })