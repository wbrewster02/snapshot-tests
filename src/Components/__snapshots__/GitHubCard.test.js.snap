// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="ripple ripple-surface bg-image hover-overlay"
    onMouseDown={[Function]}
  >
    <img
      alt="..."
      className="img-fluid"
      src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBwgHBgkIBwgKCgkLDRYPDQwMDRsUFRAWIB0iIiAdHx8kKDQsJCYxJx8fLT0tMTU3Ojo6Iys/RD84QzQ5OjcBCgoKDQwNGg8PGjclHyU3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3N//AABEIAIMAgwMBIgACEQEDEQH/xAAbAAEAAgMBAQAAAAAAAAAAAAAABQYBAwQCB//EADMQAQACAQICBggFBQAAAAAAAAABAgMEEQVREiEiMXGxFTRBYXKRodETFCMykiRCUnOB/8QAFgEBAQEAAAAAAAAAAAAAAAAAAAEC/8QAFhEBAQEAAAAAAAAAAAAAAAAAABEB/9oADAMBAAIRAxEAPwD6IA2yAAAAAAAAAAAAAADO0gMAAAAAADZhwZc9ujipNvf7HdTg+W0fqZaVnlHWVUaJK/BskR2M1Z8Y2cWfT5dPbbLSY5T7JKNQAgAAAAAAAAAA26bBbUZYx09vfPKGpL8CxdnJlmOuZ6MeCaqQ0+CmnxVx442iPq2ssMqPGXHXLjml43rL2yCsarBbT5rUt3d9Z5w0pnjeKJw0yxE71tt1cpQzWagAqAAAAAAAAEp7g3qVfilAymeB5N8N8e/XW2//ACU1UoAyoADi4tH9Dl923mr6b43k6OlikT13t9EJC4gA0gAAAAAAAA3aPPOm1EZI7u60c4aTmaq1Y71yUi9J3rPXEvat6LXZdLvEdqk/2zPclMXF9Nav6nSpPKY38mYqQeb2rSs2tO0RG8zLhycX01Y7HSvPsiI280breIZNVEU26NP8Y9pB412pnVZ+nH7Y6qw5waQAEAAAAAAAd+g4fOo2yZd4x/WwripS+S3Rx0m0+5104Xqrdc1rX4pTmLFjw1iuOkVj3NjNVBeiNTzx/wAp+zHofU88fzn7J4KIH0RqeeP5z9j0RqeeP5z9k8FFetwzVV7qRbwtDmyY8mKdslLV8YWpi1YtG1oiYnviSpFTEtr+FxtOTTRtMd+P7IlrAAEAAAJFdfDNN+Y1G9o7FOufesMRERERHVDm4dg/A0lK7bWntW8ZdTGqAAAAAAAAIXjGl/Dv+YpHZt+6PemmnV4ozafJjn216vEFYCJ3G2QAB6xx0sla87RDy2af1jF8dfMFpgBhoAAAAAAAAABU7Rte3jLD3ljbNkjleY+rw2gAINmn9YxfHXzAFpAYaAAAAAAAAGGQFX1XrWb/AGW82oG0ABH/2Q=="
    />
    <a
      href="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBwgHBgkIBwgKCgkLDRYPDQwMDRsUFRAWIB0iIiAdHx8kKDQsJCYxJx8fLT0tMTU3Ojo6Iys/RD84QzQ5OjcBCgoKDQwNGg8PGjclHyU3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3N//AABEIAIMAgwMBIgACEQEDEQH/xAAbAAEAAgMBAQAAAAAAAAAAAAAABQYBAwQCB//EADMQAQACAQICBggFBQAAAAAAAAABAgMEEQVREiEiMXGxFTRBYXKRodETFCMykiRCUnOB/8QAFgEBAQEAAAAAAAAAAAAAAAAAAAEC/8QAFhEBAQEAAAAAAAAAAAAAAAAAABEB/9oADAMBAAIRAxEAPwD6IA2yAAAAAAAAAAAAAADO0gMAAAAAADZhwZc9ujipNvf7HdTg+W0fqZaVnlHWVUaJK/BskR2M1Z8Y2cWfT5dPbbLSY5T7JKNQAgAAAAAAAAAA26bBbUZYx09vfPKGpL8CxdnJlmOuZ6MeCaqQ0+CmnxVx442iPq2ssMqPGXHXLjml43rL2yCsarBbT5rUt3d9Z5w0pnjeKJw0yxE71tt1cpQzWagAqAAAAAAAAEp7g3qVfilAymeB5N8N8e/XW2//ACU1UoAyoADi4tH9Dl923mr6b43k6OlikT13t9EJC4gA0gAAAAAAAA3aPPOm1EZI7u60c4aTmaq1Y71yUi9J3rPXEvat6LXZdLvEdqk/2zPclMXF9Nav6nSpPKY38mYqQeb2rSs2tO0RG8zLhycX01Y7HSvPsiI280breIZNVEU26NP8Y9pB412pnVZ+nH7Y6qw5waQAEAAAAAAAd+g4fOo2yZd4x/WwripS+S3Rx0m0+5104Xqrdc1rX4pTmLFjw1iuOkVj3NjNVBeiNTzx/wAp+zHofU88fzn7J4KIH0RqeeP5z9j0RqeeP5z9k8FFetwzVV7qRbwtDmyY8mKdslLV8YWpi1YtG1oiYnviSpFTEtr+FxtOTTRtMd+P7IlrAAEAAAJFdfDNN+Y1G9o7FOufesMRERERHVDm4dg/A0lK7bWntW8ZdTGqAAAAAAAAIXjGl/Dv+YpHZt+6PemmnV4ozafJjn216vEFYCJ3G2QAB6xx0sla87RDy2af1jF8dfMFpgBhoAAAAAAAAABU7Rte3jLD3ljbNkjleY+rw2gAINmn9YxfHXzAFpAYaAAAAAAAAGGQFX1XrWb/AGW82oG0ABH/2Q=="
    >
      <div
        className="mask"
        style={
          Object {
            "backgroundColor": "rgba(251, 251, 251, 0.15)",
          }
        }
      />
    </a>
  </div>
  <div
    className="card-body"
  >
    <h5
      className="card-title"
    >
      William Brewster
    </h5>
    <p
      className="card-text"
    >
      Currently enrolled in NCSU Software Development Bootcamp! Github username: wbrewster02
    </p>
  </div>
</div>
`;
